### DISCUSSION TITLE
Reverse conversion (.docx to options json)

### DISCUSSION THREAD

**Discussion by iamelru** [dolanmiu/docx#1368] [Ideas / Requests]
Do you have plans on implementing a reverse functionality that will generate a `.json` off of `.docx`? For example, it could parse `.docx` and return a class name and options as `.json`:
`{ elem: 'TextRun', text: 'Hello world', bold: true}`

Thank you!

**Comment by dolanmiu** [comment]
Sorry, nope, this is out of scope I am afraid

docx is supposed to only be one way
Similar questions have been asked for .docx to .html for example
